pvr.mythtv/changelog.txt

```diff
@@ -1,3 +1,6 @@
+v5.2.0
```

Comment:
changelog version number does not match version number in addon.xml.in (5.1.4 vs 5.2.0)